spec/generators/sorbet_test_cases.rb

```diff
@@ -305,6 +305,21 @@
 T.assert_type!(Wizard::House::Ravenclaw, Wizard::House)
 T.assert_type!(Wizard::House::Slytherin, Wizard::House)
 
+# Serialization
+T.assert_type!(
+  wizard.owl_results,
+  T.nilable(T.any(T::Hash[T.untyped, T.untyped]))
```

Comment:
```suggestion
  T.nilable(T::Hash[T.untyped, T.untyped])
```

Reply:
(I didn't know about Github's "suggestion" tools so added this in a separate commit - https://github.com/chanzuckerberg/sorbet-rails/pull/385/commits/ebbde2e95d54d7e82893594b4ae330561aa56dbd)